src/ExportGenerators.mss

```diff
@@ -973,17 +973,18 @@ function GenerateRest (bobj) {
         libmei.AddAttribute(r, 'fontsize', 'small');
     }
 
-    if (bobj.Hidden = true and name != 'space')
-    {
-        libmei.AddAttribute(r, 'visible', 'false');
-    }
-
     if (bobj.Color != 0 and name != 'space')
     {
         nrest_color = ConvertColor(bobj);
         libmei.AddAttribute(r, 'color', nrest_color);
     }
 
+    position = bobj.RestPosition;
+    if (position != 0 and name != 'space')
+    {
+        libmei.AddAttribute(r, 'loc', (position + 64) / 16);
+    }
```

Comment:
Unfortunately it's not that easy to determine the rest position. While I understand `@loc` as an absolute value (at least that's how Verovio handles it), `RestPosition` is relative to a default position. Sibelius' default position depends on if there are other voices and what is the content of those other voices. We'd have to reverse-engineer how Sibelius shifts rests.

Manual adjustments of rest positions is primarily of interest in multi-voice situations, but those are precisely the situations where we can't really tell what the rest position is.

Sorry for the hussle @rettinghaus , but could you remove this as well?

Reply:
Ok, didn't test it with multiple voices …